src/EnergyPlus/PlantUtilities.cc

```diff
@@ -1805,6 +1823,20 @@ void ScanPlantLoopsForNodeNum(EnergyPlusData &state,
     }
 }
 
+// Utility function, mostly for unit tests.  
```

Comment:
This is mostly for unit tests, where `PlantLoc` objects are filled explicitly as opposed to by calls to `ScanPlantLoopsForObject`. Once you have all of the indices, this call will fill in the right values for the pointers. One caveat, this obviously needs to be called *after* the plant object tree is created.